idea_plugin/build.gradle

```diff
@@ -15,42 +15,35 @@
  */
 
 plugins {
-  id "org.jetbrains.intellij" version "0.3.1"
-}
-
-task showProps {
-  doLast {
-    println "Your password is ${jetbrainsPluginRepoPassword}"
-  }
+  id "org.jetbrains.intellij" version "0.4.21"
 }
 
 repositories {
   mavenCentral()
 }
 
 ext {
-  googleJavaFormatVersion = '1.5'
+  googleJavaFormatVersion = '1.8'
```

Comment:
Feel like some of the gradle files here use 1.8 instead of 1.9

Reply:
Yeah, I'm not sure how they manage this. It always seems to be behind. Maybe it's contributed, or maybe it's always 1 version behind because of a pending release. In any case, this is pure upstream so I want to leave it alone.